bin/java

```diff
@@ -202,6 +208,22 @@ install_java_with_overlay() {
 	fi
 }
 
+# Deprecated legacy function to install OpenJDK, we're keeping it around to not break
+# existing buildpacks that use it. If you're implementing a new buildpack, do not use
+# this function.
+install_java_with_overlay() {
```

Comment:
(potentially for another PR) The README still suggests using this deprecated function, and `bin/compile` still uses it. Though presume you will migrate these later.

Reply:
I wanted to put it in a different PR, but with the CHANGELOG entry I decided to put the README and `bin/compile` changes in here as well. (cfbb0d232bfff847a08f5232c3241ef34585c2aa) 👍🏻